test/AutoDiff/tbdgen.swift

```diff
@@ -34,6 +34,18 @@ extension Float {
   func vjpY() -> (Float, (Float) -> Float) {
     return (.zero, { $0 })
   }
+
+  // This should generate public symbols for both JVP and VJP.
```

Comment:
This should _not_ generate public symbols because the declaration is internal. This also means that the test on line 23 is wrong. Please change the `extension` to be `public` and see if the test passes.

Reply:
I changed `extension Float` to be `public` with no other changes, and the test passes.
I assume the test comments are correct now?

Reply:
Yep! Thanks for fixing this.